validator/keymanager/v2/direct/direct.go

```diff
@@ -375,3 +421,52 @@ func (dr *Keymanager) askUntilPasswordConfirms(
 	}
 	return secretKey, password, nil
 }
+
+func inputPassword(
```

Comment:
Is this duplicated to every keymanager type?

Reply:
Just between direct and derived. It's ok to have a little duplication, as we get the benefit that these keymanagers are fully in control of how to handle the accounts password.